handlers/grafana.go

```diff
@@ -25,47 +28,72 @@ func GetGrafanaInfo(w http.ResponseWriter, r *http.Request) {
 	RespondWithJSON(w, code, info)
 }
 
-func getGrafanaService() (*v1.ServiceSpec, error) {
+func getOpenshiftRouteURL(namespace string, name string) (string, error) {
+	client, err := kubernetes.NewOSRouteClient()
```

Comment:
Minor comment:
Naming is quite hard in our profession ( https://twitter.com/mariofusco/status/971106456073375745 ), so this is not blocker, but NewOSRouteClient sounds to me very specific, I tend to guess that NewOpenShiftClient as similar as IstioClient is more general.
If tomorrow, we need to query something additional, then we don't need to refactor this.
Anyway, not a blocker, but it came to my attention while reviewing the code.

Reply:
Also, this comments applies to the extended osRouteSupplier, and other similar uses.
It sounds quite specific IMHO for clarity of the code.

Reply:
True that it is very specific and doesn't look very nice but there is actually a reason for that: if you look at kubernetes/types.go , the API group name is "route.openshift.io" so it's more specific than just OpenShift. I wanted to make it clear while consuming the client API.

Reply:
PS: funny Mario's tweet :)

Reply:
Ok, but if in the future we need to query more than the "route.openshift.io" API I guess it's good to encapsulate them in a similar "IstioClient" to hide if we are calling to one or another.